Aspire BudgetingTests/MockSpreadsheet.swift

```diff
@@ -0,0 +1,30 @@
+//
+// MockSpreadsheet.swift
+// Aspire BudgetingTests
+//
+
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/c832bf4daa051d3d91d7e9434f5fff223299bc47/Aspire BudgetingTests/MockSpreadsheet.swift#L6"></span>Limit vertical whitespace to a single empty line. Currently 2.<br />
<code>vertical_whitespace</code> <code>MockSpreadsheet.swift:6</code></td>
    </tr>
  </tbody>
</table>